manifests/params.pp

```diff
@@ -364,6 +364,33 @@
     } elsif ($::operatingsystem == 'Debian' and versioncmp($::operatingsystemrelease, '9') >= 0) {
       # Debian stretch uses a different dav_svn from Ubuntu Xenial
       $php_version = '7.0'
```

Comment:
Rather than repeating `$mod_packages`, you could add a condition to control `$php_version`:

```suggestion
      if (versioncmp($::operatingsystemrelease, '10') >= 0) {
        $php_version = '7.3'
      } else {
        $php_version = '7.0'
      }
```